spanner/spansql/parser.go

```diff
@@ -908,7 +908,7 @@ func (p *parser) eat(want ...string) bool {
 
 	for _, w := range want {
 		tok := p.next()
-		if tok.err != nil || tok.value != w {
+		if tok.err != nil || !strings.EqualFold(tok.value, w) {
```

Comment:
The reason that I noticed this one in the original PR is that everything in `spannertest` is case sensitive. Adding case insensitivity specifically only here will probably cause more confusion than be of any help.